STATUS: INLINE FAIL SETUPTEARDOWN
MODIFIED: de.strullerbaumann.visualee.examiner.cdi.ExaminerInstanceTest.testFindAndSetAttributesSetInstance
MODIFIED FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/examiner/cdi/ExaminerInstanceTest.java
CLEANER: de.strullerbaumann.visualee.filter.boundary.FilterConfiguratorTest.testSetFilterUnknown
CLEANER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/filter/boundary/FilterConfiguratorTest.java
POLLUTER: de.strullerbaumann.visualee.filter.boundary.FilterConfiguratorTest.testSetFilterConfigsSource
POLLUTER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/filter/boundary/FilterConfiguratorTest.java
ORIGINAL CLEANER SIZE: 10
NEW CLEANER SIZE: 1
ELAPSED TIME: 32.043999910354614
==========================
@@ -75,0 +75,3 @@
+ {
+     FilterContainer.getInstance().clear();
+ }
